### PR TITLE
BaSP-MR-92:form component classes

### PR DESCRIPTION
**Pull request by Eugeniagassman**
i added the form component in classes

### CODE REVIEW

**Comment by LucianoStradiot** [comment]
[Form Components - Classes](https://trello.com/c/OzRsos75/92-form-components-classes)

File: src/Components/Classes/FormClasses/index.js [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+    day: '',
+    hour: '',
+    trainer: '',
+    activity: '',
+    slots: ''
+  });
+  const [responseModal, setResponseModal] = useState({
+    title: '',
+    description: '',
+    isConfirm: false
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: you don't need it any more
```suggestion
    
```

File: src/Components/Classes/FormClasses/index.js [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+  const getClasses = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/classes`);
+      const { data: classes } = await response.json();
+      setClasses(classes);
+    } catch (error) {
+      setResponseModal({
+        title: 'ERROR!',
+        description: error.message,
+        isConfirm: false
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: you don't need it any more
```suggestion
        
```

File: src/Components/Classes/FormClasses/index.js [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+  const getActivities = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/activities`);
+      const { data: activities } = await response.json();
+      setActivities(activities);
+    } catch (error) {
+      setResponseModal({
+        title: 'ERROR!',
+        description: error.message,
+        isConfirm: false
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: same here
```suggestion
        
```

File: src/Components/Classes/FormClasses/index.js [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+  const getTrainers = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/trainers`);
+      const { data: trainers } = await response.json();
+      setTrainers(trainers);
+    } catch (error) {
+      setResponseModal({
+        title: 'ERROR!',
+        description: error.message,
+        isConfirm: false
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: same here
```suggestion
       
```

File: src/Components/Classes/FormClasses/index.js [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+          day: '',
+          hour: '',
+          trainer: '',
+          activity: '',
+          slots: ''
+        });
+        setResponseModal({
+          title: 'Success!',
+          description: createdClassData.message,
+          isConfirm: false
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: same here
```suggestion
          
```

File: src/Components/Classes/FormClasses/index.js [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+        });
+        setIsClassCreated(true);
+        setIsOpen(true);
+      }
+    } catch (error) {
+      setResponseModal({
+        title: 'ERROR!',
+        description: error.message,
+        isConfirm: false
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: same here
```suggestion
      
```

File: src/Components/Classes/FormClasses/index.js [hunk 7]
```diff
@@ -0,0 +1,311 @@
...
+          day: '',
+          hour: '',
+          trainer: '',
+          activity: '',
+          slots: ''
+        });
+        setResponseModal({
+          title: 'Success!',
+          description: updatedClassData.message,
+          isConfirm: false
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: same here
```suggestion
         
```

File: src/Components/Classes/FormClasses/index.js [hunk 8]
```diff
@@ -0,0 +1,311 @@
...
+        });
+        setIsOpen(true);
+        setIsClassCreated(true);
+      }
+    } catch (error) {
+      setResponseModal({
+        title: 'ERROR!',
+        description: error.message,
+        isConfirm: false
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: same here
```suggestion
        
```

File: src/Components/Classes/FormClasses/index.js [hunk 9]
```diff
@@ -0,0 +1,311 @@
...
+    }
+    setIsOpen(!isOpen);
+  };
+  return (
+    <div>
+      <Modal
+        title={responseModal.title}
+        desc={responseModal.description}
+        isOpen={isOpen}
+        confirmModal={responseModal.isConfirm}
```
Comment (Tilajc) [inline src/Components/Classes/FormClasses/index.js]: you can delete this too
```suggestion
        
```

**Review by Tilajc — COMMENTED** [review]
Good job! just a little comment

**Review by Tilajc — APPROVED** [review]
Good Job!

**Comment by LautaroVercellini** [comment]
Good work Euge!

**Pull request merged**